script.js

```diff
@@ -1,44 +1,53 @@
-/* eslint-disable no-unused-vars, no-use-before-define */
+/* eslint-disable no-unused-vars, no-use-before-define, class-methods-use-this */
 
 const bookArray = localStorage.getItem('books') ? JSON.parse(localStorage.getItem('books')) : [];
 localStorage.setItem('books', JSON.stringify(bookArray));
 const library = JSON.parse(localStorage.getItem('books'));
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
+class Library {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
```

Comment:
**OPTIONAL**

You can also consider adding unique `id` property for each book since it would make it easier to remove books with duplicate values. The `id` property can be anything, as long as it's unique for each book, which can later be used for finding a book :blush: